homeassistant/components/sensor/__init__.py

```diff
@@ -151,6 +160,25 @@ def capability_attributes(self) -> Mapping[str, Any] | None:
     def state_attributes(self) -> dict[str, Any] | None:
         """Return state attributes."""
         if last_reset := self.last_reset:
+            if (
+                last_reset is not None
```

Comment:
This is not needed, L162 will not continue if this is `None`

Reply:
Thanks, added to #54624